packages/graphql-tower-storage-s3/src/index.js

```diff
@@ -109,25 +110,25 @@ export default class StorageS3 {
     return toKey;
   }
 
-  async fetchCover(key, width = 1920, height = null) {
+  async fetchCover(key, format, width = 1920, height = null) {
     const cacheName = `cache/${key}_cover_${[width, height].join('x')}`;
 
     try {
       await this.s3.headObject({ Key: cacheName }).promise();
     } catch (e) {
-      await this.transform(`media/${key}_cover`, cacheName, sharp().resize(width, height).jpeg());
+      await this.transform(`media/${key}_cover`, cacheName, format, sharp().resize(width, height));
```

Comment:
`await this.transform(media/${key}_cover, cacheName, toFormat(sharp().resize(width, height), format));`